cvat-core/src/quality-report.ts

```diff
@@ -91,15 +105,16 @@ export default class QualityReport {
 
     get summary(): QualitySummary {
         return {
-            frameCount: this.#summary.frame_count,
-            frameSharePercent: this.#summary.frame_share * 100,
+            totalFrames: this.#summary.total_frames,
+            validationFrames: this.#summary.validation_frames,
+            validationFrameSharePercent: this.#summary.validation_frame_share * 100,
```

Comment:
Is there any reason to convert to `percent`? 
In accuracy/precision/recall we also multiply to 100, but we do not add any `percent` prefix

I would vote to keep these names and values API-consistent.